cmd/list.go

```diff
@@ -68,17 +69,29 @@ func listGenesis(cmd *cobra.Command, args []string) error {
 
 	for _, f := range files {
 		if strings.Contains(f.Name(), constants.Sidecar_suffix) {
+			carName := strings.TrimSuffix(f.Name(), constants.Sidecar_suffix)
 			// read in sidecar file
-			sc, err := app.LoadSidecar(strings.TrimSuffix(f.Name(), constants.Sidecar_suffix))
+			sc, err := app.LoadSidecar(carName)
 			if err != nil {
 				return err
 			}
 
+			chainID := sc.ChainID
+			// for older sidecars, check in genesis if sidecar has
+			// no chainID set
+			if chainID == "" {
+				sc, err := app.LoadEvmGenesis(carName)
+				if err != nil {
```

Comment:
on error, we probably just want to use `""` for the chainId instead of failing the command

Reply:
Applied